src/WarcraftLegacies.Source/Quests/QuestYoggSaron.cs

```diff
@@ -38,7 +38,7 @@ public QuestYoggSaron(LegendaryHero yoggsaron, unit yoggsaronPrison) : base("The
     }
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Yogg-Saron is emprisonned near Storm peaks, and can be slain to acquire X";
+    protected override string RewardDescription => "Yogg-Saron is imprisonned near Storm peaks, and can be slain to acquire Val'anyr, Hammer of Ancient Kings";
```

Comment:
```suggestion
    protected override string RewardDescription => "Gain the ability to release Yogg-Saron from his prison; he can be slain to acquire Val'anyr, Hammer of Ancient Kings";
```
`RewardDescription` should indicate exactly what happens when the quest is completed!